generators/client/templates/react/src/main/webapp/app/config/icon-loader.ts.ejs

```diff
@@ -76,6 +76,8 @@ export const loadIcons = () => {
     faFlag,
     faBell,
     faHome,
-    faRoad
+    faRoad,
+    faTimesCircle,
+    faSearch
```

Comment:
shouldn't this be conditional as it's used only when elasticsearch is used

Reply:
I don't know, I don't think it's really that important. I mean do we really need to start making that file complicated just to save a few kb. wdyt?

Reply:
I guess one icons doesn't harm that much, we can start adding conditions if there are many